fusor-ember-cli/app/templates/components/tr-deployment.hbs

```diff
@@ -1,7 +1,7 @@
-<td> {{#link-to 'deployment' deployment}} {{deployment.name}} {{/link-to}}</td>
+<td> {{#link-to routeNameForEdit deployment}} {{deployment.name}} {{/link-to}}</td>
 <td> {{deployment.lifecycle_environment.name}} </td>
 <td> {{deployment.organization.name}} </td>
-<td> {{deployment.state}} {{deployment.progressPercent}}  </td>
+<td> {{deployment.state}} {{!deployment.progressPercent}}  </td>
```

Comment:
I removed the % on the list of deployments page since it's the progress of "Deploy" task and shows 
" running 0.0%" for too long" when the sub-tasks have progress.
This wasn't a requirement anyway to show progress in the list of deployments.